libs/clients/syslumenn/src/lib/syslumennClient.utils.ts

```diff
@@ -257,12 +259,16 @@ export const mapEstateRegistrant = (
   return {
     applicantEmail: syslaData.tolvuposturSkreningaradila ?? '',
     applicantPhone: syslaData.simiSkraningaradila ?? '',
-    knowledgeOfOtherWills: syslaData.vitneskjaUmAdraErfdaskra ? 'yes' : 'no',
+    knowledgeOfOtherWills: syslaData.vitneskjaUmAdraErfdaskra ? 'Yes' : 'No',
     districtCommissionerHasWill: syslaData.erfdaskraIVorsluSyslumanns ?? false,
     assets: syslaData.eignir
       ? syslaData.eignir
-          .filter((a) => a.tegundAngalgs === TegundAndlags.NUMBER_0)
-          .filter((a) => a?.fastanumer && /^[fF]{0,1}\d{7}$/.test(a.fastanumer))
+          .filter(
+            (a) =>
+              a.tegundAngalgs === TegundAndlags.NUMBER_0 &&
```

Comment:
Is tegundAngalgs a typo? Or is it perhaps like that i the open api definition?

Reply:
This is how the definition arrives after running `yarn nx run clients-syslumenn:update-openapi-document`. It is a typo, yes, but not on our end.